README.md

```diff
@@ -42,28 +44,38 @@ To load modules, run `modprobe`. (eg: `sudo modprobe -a tcp_vegas`). You should
 
 To generate the graphs in our report, run:
 ```
-$ sh run_experiments.sh
+$ sh run_all.sh
```

Comment:
add step after this: after run_all.sh as finished its first master iteration (i.e. you see "MASTER ITERATION: 2") printed to the terminal, Ctrl+Z

Reply:
i changed it to run 5 times instead of 1000, do we still need this?

Reply:
or can we remove the loop so ctrl+z doesn't need to happen?

Reply:
You can remove the loop, and hardcode in results-1 at the bottom instead of results-$i

Reply:
And also logs-1, and remove the echo statement that says MASTER ITERATION